File: web-app/problems/two-sum/full-boilerplate/full-boilerplate.cpp
#include <iostream>
  #include <vector>
  #include <string>
  #include <sstream>
  
  int main() {
    std::vector<std::string> lines; // Assume input is loaded into lines
  
    int num1;
std::istringstream(lines[0]) >> num1;
  int num2;
std::istringstream(lines[1]) >> num2;
    int result = sum(num1, num2);
    std::cout << result << std::endl;
    return 0;
  }

File: web-app/problems/two-sum/full-boilerplate/full-boilerplate.rs
use std::fs::read_to_string;
  use std::io::{self};
  use std::str::Lines;
  
  fn main() -> io::Result<()> {
    let input = read_to_string("/path/to/input.txt")?;
    let mut lines = input.lines();
    let num1: i32 = lines.next().unwrap().parse().unwrap();
  let num2: i32 = lines.next().unwrap().parse().unwrap();
    let result = sum(num1, num2);
    println!("{}", result);
    Ok(())
  }